post-deployment/openstack/post-deployment.yml

```diff
@@ -29,5 +29,7 @@
 - import_playbook: ingress/main.yml
```

Comment:
somewhat unrelated... but should we add 
`  when: (extraGateway | default(false) | bool) or (net2 | default(false) | bool)`
here so that non-EG/Net2 clusters don't run the job to update IngressControllers?